projects/vdk-plugins/vdk-impala/tests/jobs/load_versioned_template_partition_job/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Is there any reason for the newly added content of `tests/jobs` not going to `tests/functional/jobs`, and also template_regression_test.py? Why introducing the new file structure, since jobs directory was already existing?

Reply:
I forgot to move it.